intelij-plugin/src/main/java/tech/pantheon/yanginator/plugin/formatter/settings/YangCodeStyleMainPanel.java

```diff
@@ -0,0 +1,20 @@
+package tech.pantheon.yanginator.plugin.formatter.settings;
+
+import com.intellij.application.options.TabbedLanguageCodeStylePanel;
+import com.intellij.psi.codeStyle.CodeStyleSettings;
+import tech.pantheon.yanginator.plugin.YangLanguage;
+
+public class YangCodeStyleMainPanel extends TabbedLanguageCodeStylePanel {
+    public YangCodeStyleMainPanel(final CodeStyleSettings currentSettings,
+                                  final CodeStyleSettings settings) {
+        super(YangLanguage.INSTANCE, currentSettings, settings);
+    }
+
+    @Override
+    protected void addBlankLinesTab(final CodeStyleSettings settings) {
```

Comment:
These two NO-OP methods would probably do well with a comment explaining the reason why they're empty.